### PR TITLE
Add a "Used in these app" section to the readme

### PR DESCRIPTION
**Pull request by DivineDominion**
I started bare-bones:

```md
- [TableFlip](https://tableflipapp.com/)
- [The Archive](https://zettelkasten.de/the-archive/)
- [Word Counter](https://wordcounterapp.com/)
```

How much detail do you think is interesting before it turns into an advert? 

With short description:

```md
- [TableFlip](https://tableflipapp.com/), visual Markdown table editor
- [The Archive](https://zettelkasten.de/the-archive/), note-taking application
- [Word Counter](https://wordcounterapp.com/), measuring writer's productivity
```

I would think that e.g. this would be too much:

```md
- [The Archive](https://zettelkasten.de/the-archive/), lightning-fast app to manage your knowledge and write more
```

Additionally, what about author attributions?

```md
- [TableFlip](https://tableflipapp.com/), visual Markdown table editor by [Christian Tietze][...]
- [The Archive](https://zettelkasten.de/the-archive/), note-taking application by [Christian Tietze][...]
- [Word Counter](https://wordcounterapp.com/), measuring writer's productivity by [Christian Tietze][...]
```

Which links are desirable before it, again, turns into an advert of sorts?

- requiring a link to the GitHub profile: emphasizes the open source author aspect
- allowing links to personal websites: direct traffic as a marketing tool
- allowing links to Twitter/social media

What do you think @sindresorhus?

### CODE REVIEW

**Comment by sindresorhus** [comment]
> - [TableFlip](https://tableflipapp.com/), visual Markdown table editor by [Christian Tietze][...]

This is good, but I would tweak the formatting slightly:

> - [TableFlip](https://tableflipapp.com) - Visual Markdown table editor by [Christian Tietze]()

---

> requiring a link to the GitHub profile: emphasizes the open source author aspect

Yes.

> allowing links to personal websites: direct traffic as a marketing tool

Nah. People can find that on your GitHub profile.

> allowing links to Twitter/social media

Same as the above.

File: readme.md [hunk 1]
```diff
@@ -216,6 +216,15 @@ The `PreferencesWindowController` adheres to the [Apple HIG](https://developer.a
 You might also like Sindre's [apps](https://sindresorhus.com/apps).
 
 
+### Used in these app
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
## Used in these apps
```

File: readme.md [hunk 2]
```diff
@@ -216,6 +216,15 @@ The `PreferencesWindowController` adheres to the [Apple HIG](https://developer.a
...
+
+- [TableFlip](https://tableflipapp.com/)
+- [The Archive](https://zettelkasten.de/the-archive/)
+- [Word Counter](https://wordcounterapp.com/)
+
+Want to tell the world about your app using Preferences? Open a PR!
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
Want to tell the world about your app that is using Preferences? Open a PR!
```

?

**Comment by DivineDominion** [comment]
Ok, done :)

Are you going to add your own, too?

**Comment by sindresorhus** [comment]
Yes, but the apps I'm using it in I have yet to finish and publish, so will do it when that happens.

**Pull request merged**